huey/storage.py

```diff
@@ -122,6 +122,10 @@ def flush_all(self):
 class RedisStorage(BaseStorage):
     def __init__(self, name='huey', blocking=False, read_timeout=1,
                  max_errors=1000, connection_pool=None, **connection_params):
+        redis_url = connection_params.pop('url', None)
+        if redis_url:
+            connection_pool = redis.ConnectionPool.from_url(redis_url, decode_components=True)
```

Comment:
Also, why is `url` not just a regular keyword argument?

`name='huey', blocking=False, read_timeout=1, max_errors=1000, connection_pool=None, url=None, **connection_params)`?